### PR TITLE
Fix oracle issue and find-by-delete issue

### PR DESCRIPTION
**Pull request by KyleAure**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

### CODE REVIEW

File: dev/io.openliberty.org.testcontainers/cache/projects [hunk 1]
```diff
@@ -175,7 +175,7 @@ com.ibm.ws.transaction.hadb_fat.sqlserver.2
 com.ibm.ws.transaction.hadb_fat.sqlserver.lease
 com.ibm.ws.transaction.hadb_fat.sqlserver.retriablecodes
 com.ibm.ws.wsat.migration_fat.6
-com.ibm.ws.wsat.recovery_fat.multi.6
+com.ibm.ws.wsat.recovery_fat.multi.8
```
Comment (njr-11) [inline dev/io.openliberty.org.testcontainers/cache/projects]: Updates to this file appear to be unrelated. Was it included by accident?

**Review by njr-11 — DISMISSED** [review]

**Comment by KyleAure** [comment]
#build

**Review by njr-11 — APPROVED** [review]

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
RTC Build: Filed with 3 known defects (unrelated)
Pipeline: Failed with 1 known defect (unrelated)
